_rules/image-accessible-name-23a2a8.md

```diff
@@ -139,31 +132,58 @@ The HTML `img` element has an [accessible name][] that is not empty.
 The HTML `img` element is not [marked as decorative][] and has an empty [accessible name][].
 
 ```html
-<img />
+<img src="/test-assets/shared/w3c-logo.png" />
```

Comment:
Althought this code is considered `img` element with an empty accessibile name but I guess we can improve this example by adding `alt` attribute. What do you think?
```suggestion
<img src="/test-assets/shared/w3c-logo.png" alt=""/>
```

Reply:
But that would break the example, as `alt=""` would make the image decorative, which we don't want for this.